Formula/gdu.rb

```diff
@@ -17,8 +17,6 @@ class Gdu < Formula
 
   depends_on "go" => :build
 
-  conflicts_with "coreutils", because: "both install `gdu` binaries"
-
```

Comment:
Let's remove this in a separate PR. Removing it here means we'll also need to remove it from `coreutils` but that just results in an unnecessary rebuild.

Reply:
Ah, I see. I will create second PR for that.